src/main/resources/annotations.tsv

```diff
@@ -2372,4 +2395,275 @@ loincId	loincScale	system	code	hpoTermId	isNegated	createdOn	createdBy	lastEdite
+799-7	Qn	FHIR	N	HP:0000790	true	2018-07-24T11:43:34	JGM:azhang	NA	NA	0.1	true	NA
+799-7	Qn	FHIR	H	HP:0000790	false	2018-07-24T11:43:34	JGM:azhang	NA	NA	0.1	true	NA
+13945-1	Qn	FHIR	N	HP:0002907	true	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+13945-1	Qn	FHIR	H	HP:0002907	false	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+14290-1	Qn	FHIR	N	HP:0000790	true	2018-07-24T11:43:34	JGM:azhang	NA	NA	0.1	true	NA
+14290-1	Qn	FHIR	H	HP:0000790	false	2018-07-24T11:43:34	JGM:azhang	NA	NA	0.1	true	NA
+32776-7	Ord	FHIR	POS	HP:0002907	false	2018-07-24T11:48:39	JGM:azhang	NA	NA	0.1	true	NA
+32776-7	Ord	FHIR	NEG	HP:0002907	true	2018-07-24T11:48:39	JGM:azhang	NA	NA	0.1	true	NA
+5808-1	Qn	FHIR	N	HP:0002907	true	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+5808-1	Qn	FHIR	H	HP:0002907	false	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+59830-0	Qn	FHIR	N	HP:0002907	true	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+59830-0	Qn	FHIR	H	HP:0002907	false	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+68912-5	Qn	FHIR	N	HP:0002907	true	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+68912-5	Qn	FHIR	H	HP:0002907	false	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+68902-6	Qn	FHIR	N	HP:0000790	true	2018-07-24T11:43:34	JGM:azhang	NA	NA	0.1	true	NA
+68902-6	Qn	FHIR	H	HP:0000790	false	2018-07-24T11:43:34	JGM:azhang	NA	NA	0.1	true	NA
+68903-4	Qn	FHIR	N	HP:0000790	true	2018-07-24T11:43:34	JGM:azhang	NA	NA	0.1	true	NA
+68903-4	Qn	FHIR	H	HP:0000790	false	2018-07-24T11:43:34	JGM:azhang	NA	NA	0.1	true	NA
+63378-4	Ord	FHIR	POS	HP:0012587	false	2018-07-24T12:00:47	JGM:azhang	NA	NA	0.1	true	NA
+63378-4	Ord	FHIR	NEG	HP:0012587	true	2018-07-24T12:00:47	JGM:azhang	NA	NA	0.1	true	NA
+58449-0	Qn	FHIR	N	HP:0002907	true	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+58449-0	Qn	FHIR	H	HP:0002907	false	2018-07-24T11:47:35	JGM:azhang	NA	NA	0.1	true	NA
+53358-8	Qn	FHIR	N	HP:0012587	true	2018-07-24T12:01:26	JGM:azhang	NA	NA	0.1	true	NA
+53358-8	Qn	FHIR	H	HP:0012587	false	2018-07-24T12:01:26	JGM:azhang	NA	NA	0.1	true	NA
+53360-4	Qn	FHIR	N	HP:0012587	true	2018-07-24T12:01:26	JGM:azhang	NA	NA	0.1	true	NA
+53360-4	Qn	FHIR	H	HP:0012587	false	2018-07-24T12:01:26	JGM:azhang	NA	NA	0.1	true	NA
+49505-1	Qn	FHIR	N	HP:0000790	true	2018-07-24T12:02:56	JGM:azhang	NA	NA	0.1	true	NA
+49505-1	Qn	FHIR	H	HP:0000790	false	2018-07-24T12:02:56	JGM:azhang	NA	NA	0.1	true	NA
+50226-0	Ord	FHIR	POS	HP:0000790	false	2018-07-24T12:03:02	JGM:azhang	NA	NA	0.1	true	NA
+50226-0	Ord	FHIR	NEG	HP:0000790	true	2018-07-24T12:03:02	JGM:azhang	NA	NA	0.1	true	NA
+2465-3	Qn	FHIR	N	HP:0010701	true	2018-07-24T12:13:10	JGM:azhang	NA	NA	0.1	false	check mapping for normal
+2465-3	Qn	FHIR	H	HP:0003237	false	2018-07-24T12:13:10	JGM:azhang	NA	NA	0.1	false	check mapping for normal
+2465-3	Qn	FHIR	L	HP:0004315	false	2018-07-24T12:13:10	JGM:azhang	NA	NA	0.1	false	check mapping for normal
+800-3	Ord	FHIR	POS	HP:0001981	false	2018-07-24T12:15:06	JGM:azhang	NA	NA	0.1	true	NA
```

Comment:
A couple more places that look off and a few more below.

Reply:
These are all correct. I'm doing a direct copy of the annotations.tsv file here: https://github.com/TheJacksonLaboratory/loinc2hpoAnnotation/blob/master/Data/TSVSingleFile/annotations.tsv
Tab-delimited files don't look pretty, but I don't have any control over the source.